### DISCUSSION TITLE
Hyperlink Issues with BIRT 4.13

### DISCUSSION THREAD

**Discussion by JerryInUtah** [eclipse-birt/birt#1505] [Q&A]
We moved our ERP system from MySQL5 to MySQL8 and BIRT from 4.8 to 4.13 and our Hyperlinks no longer work.  URI links do, but drill throughs don't.  Has anyone run inti the issue and found a solution?

**Comment by merks** [comment]
This 4.14 version will be released in just a few days:

https://download.eclipse.org/birt/updates/milestone/latest

You might want to test if that has a problem.  Perhaps someone will be able to replicate your problem from your terse description, but I suspect a little more detail is required...

**Comment by JerryInUtah** [comment]
I have tried 4.14 and get the same results.  When we click on a hyperlink, it appears to build the URL correctly.  We have specified pdf output, but it attempts to display the linked report in the web browser with no data displayed.  (see attached)
![BIRT Hyperlink Issue](https://github.com/eclipse-birt/birt/assets/18102360/34903093-6549-40c1-96a1-24008fd5b0d7)

**Comment by hvbtup** [comment]
It's just a wild guess, but might this be related to the security fixes @wimjongman made for the WebViewer with https://github.com/eclipse-birt/birt/pull/1112 - or @vpease's commit https://github.com/eclipse-birt/birt/commit/b3c67677d7f892e9a2d3e2917d152d1d347a42d4 ?

**Reply by JerryInUtah** [reply]
Thanks, I'll check it out.

**Comment by wimjongman** [comment]
Can you supply a small standalone report that shows the problem?

**Reply by speckyspooky** [reply]
@JerryInUtah 

I don't know in which way you have implemented the BIRT-architecture into your EPR-system.
But my question would be do we really see the new version BIRT 4.14 in your screen?

Because with BIRT 4.14 I implemented some smaller UI changes e.g. I changed the viewer icons through new icons
and on your screen I see the old version which was used till BIRT 4.13.

Below I take a pictore from your viewer and the modernized viewer of 4.14, bt it could be that you have special integration framing.
Therefore we should be sure that we talk about the same.

But nevertheless a very small example from your side would be helpfull to make a cross-test from our side. 

### Viewer 4.13 vs. 4.14
![Viewer2](https://github.com/eclipse-birt/birt/assets/41593722/cf9471df-a663-4543-a685-aed4c934cdc1)
